src/app/view-modes/image-text/image-text.component.ts

```diff
@@ -1,10 +1,14 @@
 import { Component, OnInit } from '@angular/core';
 import { DisplayGrid, GridsterConfig, GridsterItem, GridType } from 'angular-gridster2';
 import { map, shareReplay } from 'rxjs/operators';
-import { Page } from 'src/app/models/evt-models';
-import { EVTStatusService } from 'src/app/services/evt-status.service';
-import { EditionLevel } from '../../app.config';
+import { ViewerSource } from 'src/app/models/evt-polymorphic-models';
+import { AppConfig, EditionImages, EditionLevel } from '../../app.config';
+import { Page, Surface, ViewerDataType, XMLImagesValues } from '../../models/evt-models';
+import { register } from '../../services/component-register.service';
+import { EVTModelService } from '../../services/evt-model.service';
+import { EVTStatusService } from '../../services/evt-status.service';
 
+@register(Surface)
```

Comment:
Why do you connect the `Surface` to this component? `ImageTextComponent` is actually a view mode that is connected to a specific route.